Library/NSMutableAttributedString+SetFontKeepingTraits.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import UIKit
+
+extension NSMutableAttributedString {
+  func setFontKeepingTraits(to font: UIFont, color: UIColor? = nil) {
```

Comment:
There was some tomfoolery required to set the correct font after parsing the HTML string, so I added this helper.